lib/soapy_bing/ads/campaign_performance_report.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'soapy_bing/ads/report'
+require 'date'
+
+module SoapyBing
+  class Ads
+    class CampaignPerformanceReport < Report
+      attr_reader :date_range
+
+      def initialize(options)
+        super
+        @date_range = Range.new(
+          Date.parse(options.fetch(:date_start)),
+          Date.parse(options.fetch(:date_end))
```

Comment:
I noticed that too but have decided not to change it. But I think it is simple enough that I'll change it.